services/src/main/java/org/keycloak/services/HttpResponseImpl.java

```diff
@@ -51,13 +45,11 @@ public void setStatus(int statusCode) {
 
     @Override
     public void addHeader(String name, String value) {
-        checkCommitted();
```

Comment:
Removing this `checkCommitted` stuff as it's not consistent to QuarkusHttpResponse, and don't really see the need for this